.gitignore

```diff
@@ -8,6 +8,7 @@ coverage/
 .idea/
 .vscode/launch.json
 .vscode/tasks.json
+.vscode/settings.json
```

Comment:
I don't think we want to ignore this file. It's to ensure we all use the same styling plugins etc.

Reply:
The problem is if I do then i will be pushing the location of my python interpreter to all of you which will not make sense.

Reply:
Don't commit your changes to settings.json then? 😕 
We still want this file to be consistent between all of us